CHANGELOG.md

```diff
@@ -751,8 +751,10 @@ Non-backwards compatible changes
    - The records in  `Function.Structures` and `Function.Bundles` export proofs
          of these under the names `strictlySurjective`, `strictlyInverseˡ` and
          `strictlyInverseʳ`,
-   - Conversion functions have been added in both directions to
-         `Function.Consequences(.Propositional/Setoid)`.
+   - Conversion functions for the definitions have been added in both directions 
+		 to `Function.Consequences(.Propositional/Setoid)`.
+   - Conversion functions for structures have been added in 
+		 `Function.Structures.Biased`.
```

Comment:
So, it's good to see these things documented in the `CHANGELOG` (which by its very nature, is 'permanent' only to the extent that diligent users/developers can face wading back through the GitHub history; otherwise, we should regard such commentary as 'essentially ephemeral', much like this review comment ;-)), but at *some* point, we might want such observations/ux hacks to be added to a `user-guide` to the library, which we don't really have (yet)?